packages/workspace/src/command-line/dep-graph.ts

```diff
@@ -77,7 +77,6 @@ type ParsedUserOptions = {
 type OutputOptions = {
   data: string;
   shouldOpen: boolean;
-  shouldWriteToFile: boolean;
   filename?: string;
```

Comment:
Discussed with Victor, `shouldWriteToFile` didn't seem to actually be a supported use case. If you try and pass `--output` to the command it errors at the yargs level